### ISSUE TITLE
Support for bigint in ID

### ISSUE THREAD

**Issue by vwkd** [graphql/graphql-js#3913]
It's desirable to represent an ID on the server as a `bigint` instead of a `number`.

Currently, the ID scalar type supports serializing `number` to `string`, but not `bigint`.

I'd like to see support for serialization of `bigint` to `string` similar to `number`.

I think it would require adding this code to https://github.com/graphql/graphql-js/blob/d766c8eb34fb03c5ee63c51380f2b012924e970c/src/type/scalars.ts#L225-L227

```js
if (typeof coercedValue == "bigint") {
  return coercedValue.toString();
}
```

**Comment by JoviDeCroock** [comment]
This is correct, the spec supports 128-bit numbers https://spec.graphql.org/draft/#sec-ID

**Comment by yaacovCR** [comment]
This is completed in v17 by #4550
